lib/convection/control/cloud.rb

```diff
@@ -68,8 +68,8 @@ def converge(to_stack, options = {}, &block)
       end
 
       def diff(to_stack, options = {}, &block)
-        filter_deck(options, &block).each do |stack_name, stack|
-          block.call(Model::Event.new(:compare, "Compare local state of stack #{ stack_name } (#{ stack.cloud_name }) with remote template", :info))
+        filter_deck(options, &block).each do |_stack_name, stack|
+          block.call(Model::Event.new(:compare, "Compare local state of stack #{ stack.name } (#{ stack.cloud_name }) with remote template", :info))
```

Comment:
@bcall-r7 wait if we're going to use `stack.name` anyways why even return the entire hash (vs. just the `Stack` values)?